src/server/pkg/obj/google_client.go

```diff
@@ -26,7 +31,20 @@ func (c *googleClient) Writer(name string) (io.WriteCloser, error) {
 }
 
 func (c *googleClient) Walk(name string, fn func(name string) error) error {
-	return fmt.Errorf("googleClient.Walk: not implemented")
+	query := &storage.Query{Prefix: name}
+	for query != nil {
+		objectList, err := c.bucket.List(c.ctx, query)
+		if err != nil {
+			return nil
```

Comment:
Is this intentional?  I'm assuming you want to do nothing if the prefix is not found.  But this error could mean other things, like gce connection error or whatnot?  Is it possible to check for specific error types?